src/opnsense/mvc/app/views/OPNsense/Unbound/overrides.volt

```diff
@@ -26,121 +26,91 @@
-                add:'/api/unbound/settings/addHostAlias/',
-                del:'/api/unbound/settings/delHostAlias/',
-                toggle:'/api/unbound/settings/toggleHostAlias/',
-                options: {
-                    labels: {
-                        noResults: "{{ lang._('No results found for selected host or none selected') }}"
-                    },
-                    selection: true,
-                    multiSelect: true,
-                    rowSelect: true,
-                    useRequestHandlerOnGet: true,
-                    requestHandler: function(request) {
-                        let uuids = $("#grid-hosts").bootgrid("getSelectedRows");
-                        request['host'] = uuids.length > 0 ? uuids[0] : "__not_found__";
-                        let selected = $(".host_selected");
-                        uuids.length > 0 ? selected.show() : selected.hide();
-                        if (request.rowCount === undefined) {
-                            // XXX: We can't easily see if we're being called by GET or POST, assume GET uri when there's no rowcount
-                            return new URLSearchParams(request).toString();
-                        } else {
-                            return request;
-                        }
+    let grid_hosts = $("#{{formGridHostOverride['table_id']}}").UIBootgrid({
+        search:'/api/unbound/settings/searchHostOverride/',
+        get:'/api/unbound/settings/getHostOverride/',
+        set:'/api/unbound/settings/setHostOverride/',
+        add:'/api/unbound/settings/addHostOverride/',
+        del:'/api/unbound/settings/delHostOverride/',
+        toggle:'/api/unbound/settings/toggleHostOverride/',
+        options: {
+            selection: true,
+            multiSelect: false,
+            rowSelect: true,
+            formatters: {
+                "mxformatter": function (column, row) {
+                    /* Format the "Value" column so it shows either an MX host ("MX" type) or a raw IP address ("A" type) */
+                    if (row.mx.length > 0) {
+                        row.server = row.mx + ' (prio ' + row.mxprio + ')';
                     }
-                }
-            });
-
-            if (!heading_appended) {
-                $("div.actionBar").each(function(){
-                    if ($(this).closest(".bootgrid-header").attr("id").includes("alias")) {
-                        $(this).parent().prepend($('<td id="heading-wrapper" class="col-sm-2 theading-text">{{ lang._('Aliases') }}</div>'));
-                        $(this).removeClass("col-sm-12");
-                        $(this).addClass("col-sm-10");
-                        heading_appended = true;
+                    return row.server;
+                },
+                /* commands and rowtoggles added here since adding a custom formatter removes these by default for some reason */
+                "commands": function (column, row) {
```

Comment:
both the `"commands"` and `"rowtoggle"` formatter have been copied from `opnsense_bootgrid_plugin.js`. The reason for this was that specifying a custom formatter overwrites the entire `formatters` object.

I don't know if this is still the case (I imagine it is), but this probably needs a second look (i.e. merging objects). This is outside the scope of this PR, but wanted to mention it as we are moving the code around now.